extensions/amp-story/1.0/amp-story-share-menu.js

```diff
@@ -48,9 +50,9 @@ const getTemplate = (element) => {
   return htmlFor(element)`
     <div class="i-amphtml-story-share-menu i-amphtml-story-system-reset" aria-hidden="true" role="alert">
       <div class="i-amphtml-story-share-menu-container">
-        <span class="i-amphtml-story-share-menu-close-button" role="button">
+        <button class="i-amphtml-story-share-menu-close-button" role="button" aria-label="">
```

Comment:
Can we remove the `role="button"` if it's a button?

Reply:
Yes. My bad thanks for catching this.